config.yml

```diff
@@ -171,9 +171,9 @@ genesis:
           denom: upokt
       applicationList:
         - address: pokt1mrqt5f7qh8uxs27cjm9t7v9e74a9vvdnq5jva4
-          delegatee_gateway_addresses: []
+          delegatee_gateway_addresses: [pokt15vzxjqklzjtlz7lahe8z2dfe9nm5vxwwmscne4]
           service_configs:
-            - service_id: anvil
+            - service_id: "0021"
```

Comment:
tl;dr I'm changing this back

I understand why you did this, but:
1. This is legacy from Morse adopted into PATH
- I'd rather change PATH and move forward
2. If we make this change, you'll need to change a lot more (see the search result below)
- This requires a TODO_IN_THIS_PR

![Screenshot 2024-10-16 at 12 20 31 PM](https://github.com/user-attachments/assets/ca87861d-fb8f-4577-a3a3-e2bab9a5996d)

Reply:
From my understanding, Path does not know about / have "anvil" as an `EVM` service.

Reply:
I added it. PR incoming soon.